pkg/proxy/nftables/proxier.go

```diff
@@ -1660,7 +1660,7 @@ func (proxier *Proxier) syncProxyRules() {
 	conntrack.CleanStaleEntries(proxier.conntrack, proxier.svcPortMap, serviceUpdateResult, endpointUpdateResult)
 }
 
-func (proxier *Proxier) writeServiceToEndpointRules(tx *knftables.Transaction, svcPortNameString string, svcInfo *servicePortInfo, svcChain string, endpoints []proxy.Endpoint) {
```

Comment:
ah, right; in iptables we use this for a comment, but in nftables the chain name itself contains that information already so we removed the comments